releasenotes/notes/stable-hash-63389fa04cca60ab.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixes a bug causing virtualenv hashes to change during command execution, leading to installed
+    dependencies not being found
```

Comment:
```suggestion
    dependencies not being found.
```

Reply:
actually, does this even need a release note? I'm leaning towards no.

Reply:
I think it does because the bug this fixes is fairly serious.

Reply:
I thought a bit more about this and decided that it makes sense not to include a release note.